potion.install

```diff
@@ -32,14 +32,25 @@ function potion_requirements($phase) {
   if ($phase == 'install') {
     // Asserts symfony/process is installed.
     if (!class_exists('Symfony\Component\Process\Process')) {
-      $requirements['potion_process_library'] = [
+      $requirements['potion_process_component'] = [
         'description' => t('Potion requires the <a href=":component-url" target="_blank">external Symfony Process Component</a>. The recommended way of solving this dependency is using <a href=":composer-url" target="_blank">Composer</a> running the following from the command line: <br /><code>composer require symfony/process:^3.4</code>.', [
           ':component-url' => 'https://symfony.com/doc/current/components/process.html',
           ':composer-url' => 'https://getcomposer.org',
         ]),
         'severity' => REQUIREMENT_ERROR,
       ];
     }
+
+    // Asserts symfony/finder is installed.
+    if (!class_exists('Symfony\Component\Finder\Finder')) {
+      $requirements['potion_finder_component'] = [
+        'description' => t('Potion requires the <a href=":component-url" target="_blank">external Symfony Finder Component</a>. The recommended way of solving this dependency is using <a href=":composer-url" target="_blank">Composer</a> running the following from the command line: <br /><code>composer require symfony/finder:^3.4</code>.', [
+          ':component-url' => 'https://symfony.com/doc/current/components/finder.html',
+          ':composer-url' => 'https://getcomposer.org',
+        ]),
+        'severity' => REQUIREMENT_ERROR,
+      ];
+    }
```

Comment:
At line 103 when you check utilities, you missed the `/`.
When you check `!is_file` and `!is_executable`.
`
    foreach ($utilities as $utility) {
      if (!is_file($path . '/' . $utility)) {
        $not_found[$utility] = $utility;
      }
      if (!is_executable($path . '/' . $utility)) {
          $not_executable[$utility] = $utility;
      }
    }
`

Reply:
Nop I don't. If you check the form that save this config, I sanitize the path before saving.

Anyway, I will use the utility service to get the path to asserts even when people force this settings throught the settings.php file it will works.

```php
  $path = $utility->sanitizePath($gettextConfig->get('path'));
```